devlib/platform/gem5.py

```diff
@@ -127,6 +129,11 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Enable power modelling if required
+            if self.power_models_dir:
```

Comment:
You are right - the 'activation' of the power models depends on the platform scripts. Both the environment variable (POWER_MODEL_ROOT) and --power-model are specific to one platform script. There is no default method for activating power models in gem5 unfortunately.